zng/docs/spec.md

```diff
@@ -100,47 +100,55 @@ The ZNG type system comprises the standard set of primitive types like integers,
 floating point, strings, byte arrays, etc. as well as container types
 like records, arrays, and sets arranged from the primitive types.
 
-For example, a ZNG stream representing the single string "hello world"
-looks like this:
+For example, a TZNG stream representing the single string "hello world"
+may look like this:
 ```
-9:hello, world
+#35:string
+35:hello, world
 ```
-Here, the type ID is the integer "9" representing the string type
-(defined in [Typedefs](#typedefs)) and the data value "hello, world"
-is an instance of the string type.
+Here, the first line binds a tag `35` to the ZNG `string` data type
```

Comment:
I don't quite follow this paragraph... it seems to imply that in TZNG, you'd _have_ to have a new type definition to use an existing type, which I don't think is the case - it should be possible to just write `9:hello, world` (as per the previous rev).

Now to further complicate things, none of this works at all today since we require records as top level values.... which brings me to my second comment, which is that it seems potentially confusing to lead with things that aren't implemented, as the subset of readers who try off these leading examples only to get error messages might be 😕

Reply:
> I don't quite follow this paragraph... it seems to imply that in TZNG, you'd have to have a new type definition to use an existing type, which I don't think is the case - it should be possible to just write 9:hello, world (as per the previous rev).

Ooops. This comment was based on my mixing up tzng with zng, which @philrz helpfully clarified offline. Based on the spec, you cannot just write `9:hello world` in tzng (while the equivalent in zng should be ok, which is what led to my comment).